indexers/keyvalue/PostgreSQLDBMSIndexer/__init__.py

```diff
@@ -0,0 +1,144 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import pickle
+from typing import Optional
+
+from jina.executors.indexers import BaseIndexer
+
+
+class PostgreSQLDBMSIndexer(BaseIndexer):
```

Comment:
Add a TODO that this will need to subclass from the `DBMSIndexer` (when it's merged into `master`)